Samples/PowerShell/NTA.AddFlowSources.ps1

```diff
@@ -0,0 +1,48 @@
+# This sample script demonstrates how to add enabled Flow Sources with
+# required Orion node and interfaces. Interfaces are discovered with NPM 
+# discovery feature.
+# You need to be logged as a user with enabled Allow Node Management Rights.
+
+
+# Connect to SWIS
+$hostname = "swis-machine-hostname"                     # Update to match your configuration
+$username = "admin"                                     # Update to match your configuration
+$password = New-Object System.Security.SecureString     # Update to match your configuration
+$cred = New-Object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+$swis = Connect-Swis -host $hostname -cred $cred
+
+# Default engine ID
+$engineId = 1
+
+# Create Orion Node
+# Update to match your configuration
+$nodeCaption = "my.netflownode.corp"
```

Comment:
I would use RFC2606 reserved names in examples (e.g. example.com, .net, or .org).

Reply:
Fixed